.travis.yml

```diff
@@ -3,42 +3,49 @@ services: docker
 language: bash
 addons:
     apt:
-        config:
-            retries: true
```

Comment:
Why did you remove the lines? I added the lines referring `ruby/ruby`'s `.travis.yml`.
https://github.com/ruby/ruby/blob/master/.travis.yml#L120

Reply:
My IDE says this property is not allowed there.  I wasn't able to find any documentation about it.  Everything worked well without it.  I'm happy to add it back if it actually does something, or is useful :).

Reply:
Please back it. 

When "My IDE says this property is not allowed there", removing it is not good choice.

As you may know Travis is open source and written in Ruby language. In Ruby project, the core maintainers communicate with a core member on Travis project in Ruby project. As a result, the feature that is not documented, only existed in the source code can be used for Ruby project as a possibility. Ruby project uses `.travis.yml` with very advanced way.